### PR TITLE
Updated Pull request targetting [Base]

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
- Updated the `HowTo_ChangePitchAndVolume.md` file to enhance readability and clarity of the code and instructions. 
- Changes include bold formatting for method and class names, added explanations for pitch and volume concepts, `using` statements, fields for sound effect and its properties, and modifications to the `Game.LoadContent` method. 
- Also included notes on sound looping, state checking, and usage of `MathHelper.Clamp` method. 
- Extended example added for key press detection and sound effect adjustment. 
- Updated the copyright year to 2024. The subproject commit in MonoGame has been updated to `81ed391a4d0fd81e3ee4bfe2492ad9a0e7f4cfc5`.

### CODE REVIEW

File: articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md [hunk 1]
```diff
@@ -1,44 +1,89 @@
...
+    > [!NOTE]
+    > Usings are declared at the top of the file to ensure that the necessary namespaces are available to the class. The fields are declared at the top of the class to ensure that they are accessible to all methods in the class.
+
+2. In the [Game.LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent) method, set the **SoundEffectInstance** object to the return value of [SoundEffect.CreateInstance](xref:Microsoft.Xna.Framework.Audio.SoundEffect.CreateInstance).
+
+3. In the **[Game.LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent)** method, set the **SoundEffectInstance** object to the return value of **[SoundEffect.CreateInstance](xref:Microsoft.Xna.Framework.Audio.SoundEffect.CreateInstance)**. We also optionally define a variable **soundFile** to store the location of the sound file being used with the **[TitleContainer.OpenStream](xref:Microsoft.Xna.Framework.TitleContainer#Microsoft_Xna_Framework_TitleContainer_OpenStream_System_String_)** method, which is accessed with the  **using** keyword, and include a field member variable called **soundEffect**, to hold the stream.
```
Comment (SimonDarksideJ) [inline articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:36]: The text does not match the code below, consider revising

File: articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md [hunk 2]
```diff
@@ -1,44 +1,89 @@
...
+    private SoundEffectInstance soundEffectInstance;
+    private float pitch = 0.75f;
+    private float volume = 0.5f;
     ```
 
-2. In the [Game.LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent) method, set the SoundEffectInstance object to the return value of [SoundEffect.CreateInstance](xref:Microsoft.Xna.Framework.Audio.SoundEffect.CreateInstance).
+    > [!NOTE]
+    > Usings are declared at the top of the file to ensure that the necessary namespaces are available to the class. The fields are declared at the top of the class to ensure that they are accessible to all methods in the class.
+
+2. In the [Game.LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent) method, set the **SoundEffectInstance** object to the return value of [SoundEffect.CreateInstance](xref:Microsoft.Xna.Framework.Audio.SoundEffect.CreateInstance).
```
Comment (SimonDarksideJ) [inline articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:34]: I updated SoundEffectInstance to **SoundEffectInstance** to match the styling used elsewhere in the doc.  Please ensure to use consistent styling when referring to elements or classes.

File: articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md [hunk 3]
```diff
@@ -1,44 +1,89 @@
...
 
     ```csharp
-    soundfile = TitleContainer.OpenStream(@"Content\tx0_fire1.wav");
+    using Stream soundfile = TitleContainer.OpenStream(@"Content\Sound__FileName.wav");
     soundEffect = SoundEffect.FromStream(soundfile);
     soundInstance = soundEffect.CreateInstance();
     ```
 
-3. Adjust the sound to the desired level using the [SoundEffectInstance.Pitch](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Pitch) and [SoundEffectInstance.Volume](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Volume) properties.
+4. Adjust the sound to the desired level using the [SoundEffectInstance.Pitch](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Pitch) and [SoundEffectInstance.Volume](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Volume) properties.
```
Comment (SimonDarksideJ) [inline articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:44]: Text does not match the code, I suggest the code needs to change here.

File: articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md [hunk 4]
```diff
@@ -1,44 +1,89 @@
...
+    soundEffectInstance.Play();
     ```
 
-4. Play the sound using [SoundEffectInstance.Play](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Play).
+     > [!NOTE]
+     > An instance will play once, to loop the sound, you can use the **[SoundEffectInstance.IsLooped](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.IsLooped)** property to set the sound to loop. Also note that the sound will not repeat until the sound has finished playing. You can utilise the **[SoundEffectInstance.State](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.State)** property to check if the sound is playing, paused or stopped. Use the **[SoundEffectInstance.Stop](xref:Microsoft.Xna.Framework.Audio.SoundEffectInstance.Stop)** method to stop the sound.
+
+## An Extended Example
+
+ 1. Below the **[Game.Draw](xref:Microsoft.Xna.Framework.Game#Microsoft_Xna_Framework_Game_Draw_Microsoft_Xna_Framework_GameTime_)** method, create a new method called **IsKeyPressed**, which will check if a specified key is pressed and return a boolean value of true if it has been pressed.
```
Comment (SimonDarksideJ) [inline articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:56]: I fixed the API references, as they work differently to the original XNA interpreter.

Note the change from

> (xref:Microsoft.Xna.Framework.Game.Draw)

To

> (xref:Microsoft.Xna.Framework.Game#Microsoft_Xna_Framework_Game_Draw_Microsoft_Xna_Framework_GameTime_)

Browse the API docs on the website to get the Method links

**Review by SimonDarksideJ — COMMENTED** [review]
Minor updates and reviews.

Make sure to always build the site locally to watch for broken links or bad API references before submitting

- dotnet tool restore
- dotnet docfx docfx.json --serve

File: articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md (hunk 4 above)
Comment (MrValentine7777) [reply articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:56]: Which site?

Comment (SimonDarksideJ) [reply articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:56]: :O 
https://docs.monogame.net/api/index.html

Comment (MrValentine7777) [reply articles/monogame/howto/audio/HowTo_ChangePitchAndVolume.md:56]: I need a quick hand at how to make the # extension not done web anchors in 15 years... and was unable to find that anchor...

**Pull request merged**